src/operator/mkl/mkl_elementwise_copy-inl.h

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
```

Comment:
mxnet use 
1. ElementWiseSumCompute for tradintion elementwise forward
2. IdentityCompute for tradintion elementwise backward
both do not touch yet